iff --git a/flatpak/org.opencpn.OpenCPN.yaml b/flatpak/org.opencpn.OpenCPN.yaml
index 18e33d2a3..9429d5033 100644
--- a/flatpak/org.opencpn.OpenCPN.yaml
+++ b/flatpak/org.opencpn.OpenCPN.yaml
@@ -12,7 +12,7 @@
 
 app-id: org.opencpn.OpenCPN
 runtime: org.freedesktop.Platform
-runtime-version: 18.08
+runtime-version: 20.08
 sdk: org.freedesktop.Sdk
 command: opencpn.sh
